source/braille.py

```diff
@@ -959,6 +979,7 @@ def _addTextWithFields(self, info, formatConfig, isSelection=False):
 			# We only render fields that aren't at the start of their nodes for the first part of the reading unit.
 			# Otherwise, we'll render fields that have already been rendered.
 			self._skipFieldsNotAtStartOfNode = True
+		info.obj._brailleFormatFieldAttributesCache = formatFieldAttributesCache

```

Comment:
My concern is that this will clutter many NVDAObjects with empty format field caches. But I didn't see a better way. Suggestions welcome!